src/main/java/com/answerdigital/benhession/academy/answerkingweek2/controllers/OrderController.java

```diff
@@ -32,19 +32,25 @@ public OrderController(final OrderService orderService) {
     }
 
     @PostMapping
-    public ResponseEntity<Order> addOrder(@Valid @RequestBody final AddOrderRequest addOrderRequest) {
-        return new ResponseEntity<>(orderService.addOrder(addOrderRequest.address()), HttpStatus.CREATED);
+    public ResponseEntity<Order> addOrder(@RequestBody final OrderRequest orderRequest) {
```

Comment:
@Valid annotation should be added here to check if address is not Blank as per record configs, without Valid you can pass an empty string and the order will still be created.

Reply:
Hi Audrius, I've added @Valid at class-level to provide validation to all requests, not on individual parameters.

Reply:
@martintaylor1635  I am afraid that @Valid can't be used at class-level, it is used at method level. There is another annotation which is used at class-level -> @Validated. I've added it in my PR. So @Valid should be used at method-level.

Reply:
Agreed, using @Valid at class-level seems to be unstable. Will revert all controllers back to parameter-level validation.

Reply:
Fixed